Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -3843,7 +3843,7 @@
         <ScoreMake value="250"/>
         <ScoreKill value="250"/>
         <ScoreResult value="BuildOrder"/>
-        <SubgroupPriority value="17"/>
+        <SubgroupPriority value="19"/>

```

Comment:
Cyclone - one autocast ability - higher than Goliaths, lower than Marine (base stim not auto cast)?